### ISSUE TITLE
Enhanced update notifications

### ISSUE THREAD

**Issue by bergundy** [temporalio/ui#1136]

**Comment by stevekinney** [comment]
I totally agree. Do we have a mechanism for getting notifications for new SDK versions like we do for the server?

**Comment by bergundy** [comment]
Not yet, we will need to add this.

**Comment by lorensr** [comment]
something like this?

```
GET https://temporal.download/versions/latest
{ 
  "cli": "0.5.0"
  "typescript": "1.6.0"
  ...
}
```  

and the cloudflare worker would fetch latest version from the package manager? Like:

https://registry.npmjs.org/@temporalio/worker

<img width="562" alt="image" src="https://user-images.githubusercontent.com/251288/224450629-3deda7be-8f69-48f8-819b-662c78b43037.png">

**Comment by bergundy** [comment]
We'll want to integrate this with the server since it tracks the used SDK versions and we'd only want to show notifications for those.
